tests/cursor/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -208,7 +207,7 @@ thisWontBeFormatted  ( 1  ,3)
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 thisWontBeFormatted  ( 1  ,3)
 
-    thisWillBeFormatted(2, 3<|>);
+    thisWillBeFormatted(2, <|>3);
```

Comment:
None of those changes seem right. 

```js
thisWillBeFormatted    (2  ,3,  <|> )
```

If your cursor was after the 3, you don't want it to jump before the 3.